src/core/zowe/core_for_zowe_sdk/config_file.py

```diff
@@ -309,27 +310,25 @@ def load_secure_props(self) -> None:
 
         try:
             service_name = constants["ZoweServiceName"]
+            is_win32 = sys.platform == "win32"
 
-            if sys.platform == "win32":
+            if is_win32:
                 service_name += "/" + constants["ZoweAccountName"]
```

Comment:
This code could be moved inside the `_retrieve_credential` method along with other Windows specific logic 🙂